iceoryx_hoofs/test/moduletests/test_container_vector.cpp

```diff
@@ -960,6 +963,86 @@ TEST_F(vector_test, EraseOfMiddleElementCallsDTorAndMove)
     EXPECT_THAT(moveAssignment, Eq(2U));
 }
 
+TEST_F(vector_test, AccessOfNonExistingElementOnEmptyVectorLeadTermination)
+{
+    ::testing::Test::RecordProperty("TEST_ID", "31a4f0fb-31dd-4119-baba-31efab42c42b");
+    constexpr uint64_t CAPACITY{31U};
+    constexpr uint64_t DEFAULT_VALUE{69U};
+    constexpr uint64_t ACCESS_OFFSET{CAPACITY + 1U};
+
+    vector<uint64_t, CAPACITY> sut(0, DEFAULT_VALUE);
```

Comment:
Can we not use the default ctor if the vector is supposed to be empty anyway? Something like 

```vector<uint64_t, CAPACITY> sut;```

Eliminates the need for some constants. But can stay like this if no other changes are required IMO.

Reply:
@MatthiasKillat Right. I updated the tests so that to get rid of the unnecessary constants.